test/e2e/components/vl-cookie-consent-opt-in.js

```diff
@@ -8,19 +8,35 @@ class VlCookieConsentOptIn extends VlElement {
         return new VlCheckbox(this.driver, checkboxElement);
     }
 
+    async _getDescriptionElement() {
+        return this.shadowRoot.findElement(By.css('#description'));
+    }
+
+    async getDescription() {
+        const descriptionElement = await this._getDescriptionElement();
+        return descriptionElement.getText();
+    }
+
+    async getLabel() {
+        const checkbox = await this._getCheckbox();
+        return checkbox.getLabel();
+    }
+
     async isOptedIn() {
         return (await this._getCheckbox()).isChecked();
     }
 
     async optIn() {
         if (!(await (this.isOptedIn()))) {
-            (await this._getCheckbox()).click();
+            const checkbox = await this._getCheckbox();
+            await checkbox.click();
```

Comment:
Eventueel in `_click` functie steken om duplicatie met `optOut` te voorkomen.

Reply:
Done